From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Thu, 16 Jun 2022 12:27:05 +0200
Subject: [ARCHEOLOGY] Switched rockchip64 u-boot to v2022.04 (#3871)

> X-Git-Archeology: > recovered message: > * Switched rockchip64 u-boot to v2022.04
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for NanoPi M4V2
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for OrangePi 4
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for Helios64
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for clockworkpi a06
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for OrangePi R1 Plus LTS
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for OrangePi R1 Plus
> X-Git-Archeology: > recovered message: > * Adjust u-boot device tree for OrangePi R1 Plus LTS
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for Tinkerboard 2
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for RockPi S
> X-Git-Archeology: > recovered message: > * Adjust u-boot config for NanoPi R4S
> X-Git-Archeology: - Revision 26437e36c18bb09484f4150e396a1784cc6471b7: https://github.com/armbian/build/commit/26437e36c18bb09484f4150e396a1784cc6471b7
> X-Git-Archeology:   Date: Thu, 16 Jun 2022 12:27:05 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64 u-boot to v2022.04 (#3871)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
---
 configs/nanopi-r4s-rk3399_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/nanopi-r4s-rk3399_defconfig b/configs/nanopi-r4s-rk3399_defconfig
index f1776eee73..65423202ce 100644
--- a/configs/nanopi-r4s-rk3399_defconfig
+++ b/configs/nanopi-r4s-rk3399_defconfig
@@ -1,4 +1,5 @@
 CONFIG_ARM=y
+CONFIG_SKIP_LOWLEVEL_INIT=y
 CONFIG_ARCH_ROCKCHIP=y
 CONFIG_SYS_TEXT_BASE=0x00200000
 CONFIG_NR_DRAM_BANKS=1
@@ -9,6 +10,7 @@ CONFIG_DEBUG_UART_BASE=0xFF1A0000
 CONFIG_DEBUG_UART_CLOCK=24000000
 CONFIG_DEFAULT_DEVICE_TREE="rk3399-nanopi-r4s"
 CONFIG_DEBUG_UART=y
+CONFIG_SYS_LOAD_ADDR=0x800800
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-nanopi-r4s.dtb"
 CONFIG_MISC_INIT_R=y
 # CONFIG_SPL_RAW_IMAGE_SUPPORT is not set
-- 
Armbian

